addons/crm/views/utm_campaign_views.xml

```diff
@@ -5,19 +5,15 @@
         <field name="model">utm.campaign</field>
         <field name="inherit_id" ref="utm.utm_campaign_view_kanban"/>
         <field name="arch" type="xml">
+            <xpath expr="//field[@name='user_id']" position="after">
+                <field name="use_leads"/>
+            </xpath>
             <xpath expr="//div[@id='utm_statistics']" position="inside">
-                <field name="crm_lead_activated" invisible="1"/>
-                <div class="mr-3"
-                    groups="sales_team.group_sale_salesman"
-                    attrs="{'invisible': [('crm_lead_activated', '=', False)]}" title="Leads">
-                    <i class="fa fa-star text-muted"></i>
-                    <small class="font-weight-bold"><field name="lead_count"/></small>
-                </div>
                 <div class="mr-3"
                     groups="sales_team.group_sale_salesman"
-                    attrs="{'invisible': [('crm_lead_activated', '=', True)]}" title="Opportunities">
+                    t-att-title="record.use_leads.raw_value ? 'Leads' : 'Opportunities'">
```

Comment:
It is not really correct as both are computed in the read group it seems ... Did you have a PO final word on it ? Like Opportunities or Leads & Opportunities maybe ?

Reply:
On that front, I kept the same behavior as before which has been re-approved by LNA. Maybe one day we'll decide to actually use two fields with correct values by using the flag but as of now they do not want that. Also, if we changed it there, we would also have to change it in the stat buttons of the forms which would just not work due to the length of the label.